mappings/net/minecraft/KeyBindingRenameFix.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/class_5404 net/minecraft/KeyBindingRenameFix
```

Comment:
Maybe this could be renamed more generally to `GameOptionRenameFix`? (And the field names renamed accordingly)